hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeLayoutVersion.java

```diff
@@ -89,7 +89,11 @@ public static boolean supports(final LayoutFeature f, final int lv) {
     APPEND_NEW_BLOCK(-62, -61, "Support appending to new block"),
     QUOTA_BY_STORAGE_TYPE(-63, -61, "Support quota for specific storage types"),
     ERASURE_CODING(-64, -61, "Support erasure coding"),
-    EXPANDED_STRING_TABLE(-65, -61, "Support expanded string table in fsimage");
+    EXPANDED_STRING_TABLE(-65, -61, "Support expanded string table in fsimage"),
+    BLOCK_STORAGE_POLICY_INCLUDING_ALL_NVDIMM(-66, -66, "Support block " +
+        "Storage policy including ALL_NVDIMM"),
+    QUOTA_BY_STORAGE_TYPE_INCLUDING_NVDIMM(-67, -66, "Support quota for " +
+        "specific storage types including NVDIMM");
```

Comment:
Since there is no release after this feature is committed, No need to increase layout version twice for same feature.
Usually one version increase is sufficient per release. But for clarity separate feature can be added.

In this case feature is only one. NVDIMM SUPPORT.